subsystem/elevator.py

```diff
@@ -0,0 +1,74 @@
+import wpilib
+from robotpy_toolkit_7407 import Subsystem
+from robotpy_toolkit_7407.motors.revmotor import SparkMax
+from robotpy_toolkit_7407.utils.units import meters
+from robotpy_toolkit_7407.sensors.limit_switches.limit_switch import LimitSwitch
+
+import constants
+import config
+#importing packages
+
+
+class Elevator(Subsystem): #elevator class
+    extendMotor: SparkMax = SparkMax(config.elevator_motor_extend_id)#motor that extends the arm
+    rotationMotorRight: SparkMax = SparkMax(config.elevator_motor_turn_right_id, inverted = True) #motor that rotates the arm
+    rotationMotorLeft: SparkMax = SparkMax(config.elevator_motor_turn_left_id, inverted = False) #motor that rotates the arm
+    initialized: bool = False
+    extend_sensor = LimitSwitch(9) #senses when extension is at zero
```

Comment:
```suggestion
    mag_sensor_extend = LimitSwitch(9) #senses when extension is at zero
```
Also please make sure that if it is a Magnetic limit switch, you use the Magnetic Limit Switch class from the toolkit.

Reply:
I believe I created them separately in case we needed to add something to different types of limit switches in the future, but if you think it's unnecessary I can get rid of them.